README.md

```diff
@@ -1,96 +1,117 @@
 # Swift Numerics
   
 ## Introduction
-Swift Numerics provides a set of modules that support numerical computing in Swift.
-These modules fall broadly into two categories:
+
+Swift Numerics provides a set of modules that support numerical computing in Swift. These modules fall broadly into two categories:
 
 - API that is too specialized to go into the standard library, but which is sufficiently general to be centralized in a single common package.
 - API that is under active development toward possible future inclusion in the standard library.
 
-There is some overlap between these two categories, and API that begins in the first category may migrate to the second as it matures and new uses are discovered.
+There is some overlap between these two categories, and an API that begins in the first category may migrate into the second as it matures and new uses are discovered.
+
+Swift Numerics modules are fine-grained. For example, if you need support for Complex numbers, you can import ComplexModule¹ as a standalone module:
```

Comment:
```suggestion
Swift Numerics modules are fine-grained.
For example, if you need support for Complex numbers, you can import ComplexModule¹ as a standalone module:
```

Reply:
Thanks @stephentyrone - moving this sentence to the next paragraph LGTM

Reply:
To be clear, this is _not_ a new paragraph. New paragraphs only occur in markdown on a double line break.

Reply:
@stephentyrone Got it!